plugins/win-capture/game-capture.c

```diff
@@ -1614,7 +1644,7 @@ static void game_capture_tick(void *data, float seconds)
 		return;
 
 	} else if (!gc->showing) {
-		gc->retry_time = 10.0f;
+		gc->retry_time = 10.0f*hook_rate_to_float(gc->config.hook_rate);
```

Comment:
Style: You need to surround math operators by a single space on each side.  Doing this will exceed the line length limit, so also line-break this line.